File: sdk/MQTT/PHP/producer.php
<?php
/**
 * File: procuder.php
 * Description: This is the MQTT producer publishes a certain number
 *     of test messages to a particular topic through MQTT broker.
 *     It will first ask user for the quantity of messages.
 *
 * Author: Eamin Zhang
 * robomq.io (http://www.robomq.io)
 */

$server = "hostname";
$port = "1883";
$vhost = "yourvhost";
$username = "username";
$password = "password";
$topic = "test/any";

try {
	$client = new Mosquitto\Client("2", true); //clientid auto-assigned, clean_session=true
	$client->setCredentials($vhost.":".$username, $password);
	$client->connect($server, $port, 60); //keepalive=60

	echo "Quantity of test messages: ";
	$msgNum = rtrim(fgets(STDIN), PHP_EOL);
	for ($i = 1; $i <= $msgNum; $i++) {
		$message = "test msg ".$i;
		$client->publish($topic, $message, 1, false); //publish test messages to the topic
		$client->loop(); //frequently loop to to keep communications with broker
		sleep(1);
	}
	$client->disconnect();
} catch (Exception $e) {
	echo $e;
}
?>

File: sdk/MQTT/Python/producer.py
"""
 * File: procuder.py
 * Description: This is the MQTT producer publishes a certain number
 *     of test messages to a particular topic through MQTT broker.
 *     It will first ask user for the quantity of messages.
 *
 * Author: Eamin Zhang
 * robomq.io (http://www.robomq.io)
"""

import time
import paho.mqtt.client as mqtt

server = "hostname"
port = 1883
vhost = "yourvhost"
username = "username"
password = "password"
topic = "test/any"

try:
	client = mqtt.Client(client_id="", clean_session=True, userdata=None, protocol="MQTTv31")
	client.username_pw_set(vhost + ":" + username, password)
	client.connect(server, port, keepalive=60, bind_address="")	#connect
	client.loop_start()	#start loop
	msgNum = int(input("Quantity of test messages: "))
	for i in range(msgNum):
		message = "test msg " + str(i + 1)
		client.publish(topic, payload=message, qos=1, retain=False)	#publish
		time.sleep(1)
	client.loop_stop()	#stop loop
	client.disconnect()
except Exception, e:
	print e